### PR TITLE
The project uses TYPO3 11.4 now

### PR DESCRIPTION
**Pull request by epx-anpf**
This includes a compare of the database and a fresh version. No new content. Only the structure was compared and fixed.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,32 +18,31 @@
 	"license": "GPL-2.0-or-later",
 	"config": {
 		"platform": {
-			"php": "7.4"
+			"php": "7.4.1"
```
Comment (albig) [inline composer.json]: ??
```suggestion
			"php": "7.4"
```

File: config/sites/profile-service/config.yaml [hunk 2]
```diff
@@ -7,9 +7,10 @@ baseVariants:
     base: 'https://ddev-slub-profile-service.ddev.site'
     condition: 'applicationContext == "Development"'
 languages:
-  - title: Deutsch
+  -
+    title: Deutsch
```
Comment (albig) [inline config/sites/profile-service/config.yaml:11]: We need English, too.

**Review by albig — CHANGES_REQUESTED** [review]
Two tiny hints.

**Comment by epx-anpf** [comment]
Ok, i agree. It is ok for me how you suggested.

**Review by albig — APPROVED** [review]

**Pull request merged**